const notFoundError = (res) => {
    return res
        .status(404)
        .send("Este dado n√£o foi encontado no Banco de Dados.");
};

module.exports = {
    notFoundError,
};
